plugins/tuna-obs/back.js

```diff
@@ -0,0 +1,40 @@
+
+const { ipcRenderer } = require("electron");
+const registerCallback = require("../../providers/song-info");
+
+const fetch = require('node-fetch');
+
+const post = (data) => {
+	const port = 1608;
+	headers = {'Content-Type': 'application/json',
+		'Accept': 'application/json',
+		'Access-Control-Allow-Headers': '*',
+		'Access-Control-Allow-Origin': '*'}
+	const url = `http://localhost:${port}/`;
+	fetch(url, {method: 'POST', headers, body:JSON.stringify({data})});
+}
+
+
+
+module.exports = async (win) => {
+
+	registerCallback((songInfo) => {
+
+		// Register the calilback
```

Comment:
nit
```suggestion
		// Register the callback
```

Reply:
sorry about the typo, it's a "i" from vim that missed